homeassistant/components/logbook.py

```diff
@@ -97,6 +118,14 @@ def humanify(events):
             if event.event_type == EVENT_STATE_CHANGED:
                 entity_id = event.data['entity_id']
 
+                # This should be rafactored so that the SQL query does
+                # the filtering, possibly by adding an option entity_id
+                # field to events
```

Comment:
^^ what this comment says. Best way to achieve this is to add a column to the states table that contains a FK pointing at the events table. Out of scope for this PR so I've added it to [Pivotal](https://www.pivotaltracker.com/story/show/94254068).